services/app-api/serverless.yml

```diff
@@ -21,43 +20,26 @@ plugins:
 custom:
   stage: ${opt:stage, self:provider.stage}
   region: ${opt:region, self:provider.region}
-  iamPath: ${ssm:/configuration/${self:custom.stage}/iam/path~true, ssm:/configuration/default/iam/path~true, "/"}
-  iamPermissionsBoundaryPolicy: ${ssm:/configuration/${self:custom.stage}/iam/permissionsBoundaryPolicy~true, ssm:/configuration/default/iam/permissionsBoundaryPolicy~true, ""}
   serverlessTerminationProtection:
-    stages: # This is a list of common names for important envs that should not be destroyed.  You can remove the stage names your project doesn't use; this list is meant to be inclusive.
+    stages:
       - master
       - val
       - production
-      - develop
-      - main
-      - impl
-      - val
-      - prod
   tableName: ${env:AMENDMENTS_TABLE_NAME, cf:database-${self:custom.stage}.AmendmentsTableName}
   tableArn: ${env:AMENDMENTS_TABLE_ARN, cf:database-${self:custom.stage}.AmendmentsTableArn}
   atomicCounterTableName: ${env:AMENDMENTS_COUNTER_TABLE_NAME, cf:database-${self:custom.stage}.AmendmentsAtomicCounterTableName}
   atomicCounterTableArn: ${env:AMENDMENTS_COUNTER_TABLE_ARN, cf:database-${self:custom.stage}.AmendmentsAtomicCounterTableArn}
-
+  webAclName: ${self:service}-${self:custom.stage}-webacl
+  associateWaf:
+    name: ${self:custom.webAclName}
+    version: V2
   warmup:
-    enabled: true
-    vpc: false
-    events:
-      - schedule: rate(${ssm:/configuration/${self:custom.stage}/warmup/schedule~true, ssm:/configuration/default/warmup/schedule~true, "4 minutes"})
-    timeout: 20
-    prewarm: true
-    concurrency: ${ssm:/configuration/${self:custom.stage}/warmup/concurrency~true, ssm:/configuration/default/warmup/concurrency~true, 5}
-    folderName: node_modules/serverless-bundle/src/_warmup
-    cleanFolder: false
-  scripts:
-    hooks:
-      package:setupProviderConfiguration: |
-        # Remove the timestamp line from the warmup handler so a new function is uploaded
-        #   only if a configuration has truly changed.
-        set -e
-        if [ -d "node_modules/serverless-bundle/src/_warmup" ]; then
-          cd node_modules/serverless-bundle/src/_warmup
-          sed '/Generated/d' index.js > index.js.sub && mv -f index.js.sub index.js
-        fi
+    default:
```

Comment:
By upgarding to the lastest warmup plugin (via the helper) our configuration is simplified.  The helper plugin makes this idempotent, and the script hook can be removed.